dspace/config/spring/rest/projections.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.springframework.org/schema/beans
+           http://www.springframework.org/schema/beans/spring-beans-2.5.xsd">
+
+    <bean class="org.dspace.app.rest.projection.SpecificLevelProjection">
+        <property name="maxEmbed" value="5" />
```

Comment:
Can we make this editable in the `rest.cfg` file, perhaps alongside `projections.full.max`?  Remember, any config can be referenced in Spring XML, so we could change this to be something like :
```
 <property name="maxEmbed" value="${rest.projection.specificLevel.maxEmbed}" />
```